datafusion/core/src/physical_optimizer/projection_pushdown.rs

```diff
@@ -159,13 +159,14 @@ fn try_swapping_with_csv(
     projection: &ProjectionExec,
     csv: &CsvExec,
 ) -> Option<Arc<dyn ExecutionPlan>> {
+    // info!("csv exec: {}", csv);
```

Comment:
```suggestion
```
This should be removed.

Reply:
eagle eye! 
![image](https://github.com/apache/arrow-datafusion/assets/61894094/a08cb71c-1a36-46a5-a54a-f9e6c21fd0b6)